lxc/list.go

```diff
@@ -432,6 +432,8 @@ func (c *cmdList) parseColumns(clustered bool) ([]column, bool, error) {
 		'F': {i18n.G("BASE IMAGE"), c.baseImageFullColumnData, false, false},
 		'l': {i18n.G("LAST USED AT"), c.LastUsedColumnData, false, false},
 		'm': {i18n.G("MEMORY USAGE"), c.memoryUsageColumnData, true, false},
+		'%': {i18n.G("MEMORY USAGE%"), c.memoryUsagePercentColumnData, true, false},
+		'U': {i18n.G("CPU USAGE"), c.cpuUsageSecondsColumnData, true, false},
```

Comment:
Should be moved down (we're trying to keep alphabetical order) and probably just renamed to `u`.